packages/core/logging/core-logging-server-internal/src/logging_config.ts

```diff
@@ -63,6 +69,7 @@ export const config = {
       }),
       level: levelSchema,
     }),
+    browser: browserConfig,
```

Comment:
I'm exposing the browser logging configuration via the `logging.browser` config prefix. So the browser-side config is "included" in the server-side logging prefix that is `logging`.

I wasn't sure, but in the end I still think it makes sense to have a single prefix (and that it isn't worth introducing heavy changes to move the server-side logging under the `logging.server` prefix either), but please tell if you think otherwise or have another idea.

Reply:
It would be nice to have them symmetrical, but I think `logging.browser.*` is fine